packages/eth-json-rpc-provider/src/safe-event-emitter-provider.ts

```diff
@@ -64,12 +64,6 @@ export class SafeEventEmitterProvider extends SafeEventEmitter {
   constructor({ engine }: { engine: JsonRpcEngine }) {
     super();
     this.#engine = engine;
-
-    if (engine.on) {
-      engine.on('notification', (message: string) => {
```

Comment:
Your PR seems to imply that this isn't in use. Have you tested the extension with these lines stripped out?

Reply:
We searched for examples in all of our repos. We found a couple of uses, but only in test helpers.

I don't think it has been tested, that is a good idea though. We could probably delete this quickly in a patch, put it up as a draft, then see if CI passes.